client/elife-theme/src/elements/TextField.js

```diff
@@ -0,0 +1,44 @@
+import { css } from 'styled-components'
+import { th } from '@pubsweet/ui-toolkit'
+
+/**
+ * TODO
+ * this should be removed once it's implemented in pubsweet
+ * right now they don't have colorWarning
+ */
+const borderColor = ({ theme, validationStatus = 'default' }) =>
+  ({
+    error: theme.colorError,
+    success: theme.colorSuccess,
+    warning: theme.colorWarning,
+    default: theme.colorBorder,
+  }[validationStatus])
+
+/**
+ * TODO
+ *
+ * font-size & line-height might change to fontSizeBase in the future
+ *
+ * for some reason th didn't work for spacing
+ * padding: ${th('space.2') + 'px'};
+ * margin-bottom: ${th('space.1') + 'px'};
+ *
+ * move root margin-bottom at page level
+ */
+export default {
+  Input: css`
+    border-color: ${borderColor};
+    color: ${th('colorText')};
+    font-family: ${th('fontInterface')};
+    font-size: 16px;
+    line-height: 24px;
+    padding: ${props => `${props.theme.space[2]}px`};
+    height: ${props => `${props.theme.space[5]}px`};
+  `,
+  Label: css`
+    margin-bottom: ${props => `${props.theme.space[1]}px`};
```

Comment:
these will need updating for Tam's pr

Reply:
Are we not using `grid-styled` for layout & therefore wouldn't be using `margin-bottom` but using flexbox for spacing between elements?

Stangely, [`Root` is currently already a flexbox](https://gitlab.coko.foundation/pubsweet/pubsweet/blob/master/packages/ui/src/atoms/TextField.js#L5-7) but pubsweet is not using grid-styled for this :confused:

Reply:
You can see what I mean if you have a look at what I started doing for `TextArea`: https://github.com/elifesciences/elife-xpub/pull/259

Reply:
Updated to use theme helper. @jsms90 my understanding was that we will do margins at page level in a different issue/PR. I initially added those margins there because it was looking weird without them but now for some reason (one of these PRs) there is a huge margin of 24px after each group of Input + Label.